apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/util/IOUtils.java

```diff
@@ -18,16 +18,17 @@
 
 package org.apache.skywalking.apm.agent.core.util;
 
-/**
- * Copied from {@link org.apache.commons.io.IOUtils}
- */
-
 import java.io.ByteArrayOutputStream;
 import java.io.Closeable;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 
+/**
+ * Copied from commons-io-2.2 {@link org.apache.commons.io.IOUtils}
+ * Origin license: http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This is not necessary, but OK. The key is License in the root folder of SkyWalking. We need to follow the License requirement, which require we add the explicit statement in the end of the license file.